docs/kb/logging.md

```diff
@@ -5,19 +5,29 @@ keywords: kdb+, log, logging, q, replication, recovery
 ---
 # Using log files: logging, recovery and replication
 
+## Overview
+
+Software or hardware problems can cause a kdb+ server process to fail, possibly resulting in loss of data not saved to disk at the time of the failure. A kdb+ server can use logging of updates to avoid data loss when failures occur.
```

Comment:
```suggestion
Software or hardware problems can cause a kdb+ server process to fail, possibly resulting in loss of data if not saved to disk at the time of the failure. A kdb+ server can use logging updates to avoid data loss when failures occur.
```